data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,785 @@
+# Copyright (c) 2022 by Saugia
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Power:
+
+outfit "R01 Skirmish Battery"
+	plural "R01 Skirmish Batteries"
+	category Power
+	licenses
+		"Gegno Civilian"
+	cost 17500
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description `Skirmish batteries are very common among ships across Gegno space. They can be either a cheaper and more convenient storage of battery power in place of upgrading to larger Gegno generators, or a nice addition alongside them where more energy may be needed.`
+
+
+outfit "R02 Battlezone Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R02 Battlezone Batteries"
+	cost 32000
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description `Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a moderate amount of power for long lasting engagements. They also fit nicely on larger vessels as a buffer to power their larger systems.`
+
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description `Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage.`
+
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description `As the largest battery mounted on Gegno ships, the Crusade Battery holds a large amount of energy, more that it would take a small generator a decent amount of time to produce in order to support the energy hungry warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`
```

Comment:
Okay that these batteries dont scale but mb they could have smth added to them when they get biggger? To do a bit of a niche? Or are these meant to kinda suck except for their cost? In that case maybe this bigger one could be cheaper per outfit space or smth.

Rn this one requires more licenses but doesnt bring anything on top of the other one.